dashboard/app/controllers/registrations_controller.rb

```diff
@@ -9,6 +11,9 @@ def new
     super
   end
 
+  #
+  # PUT /users
+  #
   def update
     return head(:bad_request) if params[:user].nil?
     current_user.reload # Needed to make tests pass for reasons noted in registrations_controller_test.rb
```

Comment:
I realize this is not part of your changes, but is this still needed after converting the test to an integration test?

Reply:
It is (for now) on the `:update` route because I haven't converted the existing `registrations_controller_test.rb` to an `IntegrationTest`, only the new test files I'm introducing in this PR.